### PR TITLE
Created Global Info Capabilities

### PR DESCRIPTION
**Pull request by Abastro**
Re-opening this with rationale on the issue #4309 .

### CODE REVIEW

**Comment by Abastro** [comment]
So was the rationale not sufficient?

File: patches/minecraft/net/minecraft/world/storage/WorldInfo.java.patch [hunk 1]
```diff
@@ -41,15 +75,27 @@
...
          p_76064_1_.func_74782_a("DimensionData", nbttagcompound1);
-@@ -708,6 +711,7 @@
+@@ -387,6 +399,11 @@
+         {
+             p_76064_1_.func_74782_a("Player", p_76064_2_);
+         }
++
++        if (this.capabilities != null)
++        {
++            p_76064_1_.func_74782_a("capabilities", capabilities.serializeNBT());
```
Comment (JamiesWhiteShirt) [inline patches/minecraft/net/minecraft/world/storage/WorldInfo.java.patch]: TileEntity, ItemStack and Entity capabilities (and upcoming Chunk capabilities) use the key `"ForgeCaps"`. World seems to be the odd one out with `"capabilities"`. You should probably stick to `"ForgeCaps"` for consistency.

**Review by JamiesWhiteShirt — CHANGES_REQUESTED** [review]

File: patches/minecraft/net/minecraft/world/storage/WorldInfo.java.patch (hunk 1 above)
Comment (Abastro) [reply patches/minecraft/net/minecraft/world/storage/WorldInfo.java.patch]: Oh wait, I forgot it completely. Thanks!

**Review by JamiesWhiteShirt — APPROVED** [review]

**Review by Landmaster — APPROVED** [review]

**Comment by Abastro** [comment]
bump.

**Pull request closed without merging**